### PR TITLE
first-commit-message

### PR DESCRIPTION
**Pull request by sablemadison**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,262 @@
...
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
 
+@media(max-width: 500px) {
+    .container {
+      width: 500px;
+    }
+
```
Comment (LarynaB) [inline css/index.css:37]: You need to close your code bracket for your @media query.

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,262 @@
...
+    @media(max-width: 800px) {
+        .container {
+          width: 800px;
+        }
+
```
Comment (LarynaB) [inline css/index.css:42]: This media query as well needs to a closing bracket

Comment (larynab) [inline css/index.css]: @media(max-width: 800px) {
        .container {
          width: 800px;
        }
  }

File: css/index.css (hunk 1 above)
Comment (LarynaB) [reply css/index.css:37]: @media(max-width: 500px) {
    .container {
      width: 500px;
    }
}

File: css/index.css (hunk 2 above)
Comment (LarynaB) [reply css/index.css:42]: @media(max-width: 800px) {
        .container {
          width: 800px;
        }
  }

File: css/index.css [hunk 3]
```diff
@@ -1,3 +1,262 @@
...
+}
+
+.services-content .services-info .facts p {
+  margin-bottom: 20px;
+}
+
+.services-content .services-info .services-info-img {
+  width: 420px;
+  height: 300px;
+}
```
Comment (LarynaB) [inline css/index.css:262]: it is nicer to leave your @media queries as the bottom for regular css sheets.

Comment (LarynaB) [reply css/index.css:262]: /* Tablet */
@media (max-width: 800px) {

    .container {
        width: 100%;
    }

    header {
        flex-wrap: wrap;
    }
  
    header nav {
        width: 768px;
        padding: 0 30px;
    }

    header .logo {
        order: -1;
        display: block;
        margin: 30px auto 0;
    }

    .cta {
        justify-content: center;
    }
    
    .cta .cta-text {
        margin: 0;
    }

    .cta img {
        display: none;
    }

    .main-content .middle-img {
       width: 100%;
    }

    .main-content {
       margin: 75px 20px 35px;
    }

    .text-content {
        margin: 0 10px;
    }

    .contact {
        margin: 0 30px;
    }
}

/* Phone */
@media (max-width: 500px) {

    header {
        justify-content: center;
    }

    header nav {
        width: 100%;
        flex-direction: column;
        margin: 0;
        padding: 0;
    }

    header nav a {
        width: 100%;
        display: block;
        text-align: center;
        padding: 20px 0;
        font-size: 20px;
        border-bottom: 1px solid gray;
    }

    header nav a:first-child {
        border-top: 1px solid gray;
    }

    header nav a:hover {
        background: gray;
    }

    header .logo {
        margin: 20px 0;
    }

    .cta {
        margin-top: 50px;
        flex-flow: column;
        align-items: center;
    }

    .cta img {
        max-width: 80%;
    }

    .main-content .top-content {
        justify-content: space-around;
        flex-direction: column;
        height: 300px;
    }

    .main-content .bottom-content {
        flex-wrap: wrap;
        align-content: space-between;
        height: 440px;
    }

    .text-content {
        margin: 0;
    }

    .contact {
        margin: 20px;
    }
}

**Review by LarynaB — APPROVED** [review]
## Great
attempted media queries
## Requested Improvements
finish using both phone and tablet media queries
## Questions
Have you tried to time manage your projects such as asking help for a current problem and moving on to a next section to work on while you wait?
## Rating: {1-3}
1- need to complete media and tablet views

**Pull request merged**